app/src/main/java/com/sun/premierleague/utils/ContextExtensions.kt

```diff
@@ -0,0 +1,35 @@
+package com.sun.premierleague.utils
+
+import android.content.Context
+import android.net.ConnectivityManager
+import android.widget.ImageView
+import android.widget.Toast
+import com.bumptech.glide.Glide
+import com.bumptech.glide.request.RequestOptions
+import com.sun.premierleague.R
+
+fun Context.showToast(msg: String, duration: Int = Toast.LENGTH_SHORT) {
+    Toast.makeText(this, msg, duration).show()
+}
+
+fun Context.loadImage(urlImage: String, imageView: ImageView) {
+    Glide.with(this)
+        .load(urlImage)
+        .error(R.drawable.ic_football_error)
+        .into(imageView)
+}
+
+fun Context.loadImage(drawable: Int, imageView: ImageView) {
+    Glide.with(this)
+        .load(drawable)
+        .error(R.drawable.ic_football_error)
+        .into(imageView)
+}
```

Comment:
Sao lại context loadImage nhỉ ? 
2. Lấy context từ Imageview => Viết trong ViewExt.kt
```suggestion
fun ImageView.loadImage(data: String) {
    Glide.with(context)
        .load(data)
        .error(R.drawable.ic_football_error)
        .into(imageView)
}

fun Context.loadImage(drawable: Int, imageView: ImageView) {
    Glide.with(this)
        .load(drawable)
        .error(R.drawable.ic_football_error)
        .into(imageView)
}
```